openfl/component/aggregator/aggregator.py

```diff
@@ -881,6 +881,14 @@ def _log_big_warning(self):
             f' WARNED!!!'
         )
 
+    def stop(self, failed_collaborator: str = None) -> None:
+        """Stop aggregator execution."""
+        self.logger.info('Force stopping the aggregator execution.')
+        self.authorized_cols.remove(failed_collaborator)
+        for collaborator_name in self.authorized_cols:
```

Comment:
Maybe we shouldn't delete failed collaborator from `authorized_cols`?
alternative:
```python
for collaborator_name in filter(lambda c: c != failed_collaborator, self.authorized_cols):
    ...
```

Reply:
Make sense. Done.